src/main/resources/se/diabol/jenkins/pipeline/DeliveryPipelineView/help-maxNumberOfJobs.html

```diff
@@ -0,0 +1,3 @@
+<div>
+    Max number of Jobs to display on the page.
```

Comment:
I suggest updating to a more descriptive text, something along the lines with: 
"Limits the number of pipelines shown in the view, regardless of how many pipelines are configured"